### PR TITLE
Fix Streaming Integrator Doc link not mentioned in the Service Catalog view

### PR DESCRIPTION
**Pull request by Jayanie**
## Purpose

- This PR fixes https://github.com/wso2/product-apim/issues/10851
- This PR adds a button to navigate to the documentation related to streaming integrator

## Approach

Screenshot from 2021-04-19 07-58-57

### CODE REVIEW

**Comment by Jayanie** [comment]
Workflow: https://github.com/Jayanie/carbon-apimgt/actions/runs/762102658

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/ServiceCatalog/Listing/Onboarding.jsx [hunk 1]
```diff
@@ -101,25 +101,43 @@ function Onboarding() {
...
                         color='primary'
                         href='https://apim.docs.wso2.com/en/4.0.0/design/create-api/create-an-api-using-a-service/'
+                        endIcon={<LaunchIcon style={{ fontSize: 15 }} />}
+                    >
+                        <FormattedMessage
+                            id='ServiceCatalog.Listing.Onboarding.learn.link.micro.integrator'
+                            defaultMessage='Micro Integrator'
+                        />
+                    </Button>
+                    &nbsp;&nbsp;
```
Comment (tharikaGitHub) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/ServiceCatalog/Listing/Onboarding.jsx:123]: Shall we avoid using &nbsp; here and use {' '} instead? &nbsp; does not seem to be working in some browsers.

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/ServiceCatalog/Listing/Onboarding.jsx [hunk 2]
```diff
@@ -101,25 +101,43 @@ function Onboarding() {
...
                         endIcon={<LaunchIcon style={{ fontSize: 15 }} />}
                     >
                         <FormattedMessage
-                            id='ServiceCatalog.Listing.Onboarding.learn.link'
-                            defaultMessage='Get Started'
+                            id='ServiceCatalog.Listing.Onboarding.learn.link.streaming.integrator'
+                            defaultMessage='Streaming Integrator'
                         />
                     </Button>
+                    &nbsp;&nbsp;
```
Comment (tharikaGitHub) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/ServiceCatalog/Listing/Onboarding.jsx:140]: Let's use {' '} instead.

**Review by tharikaGitHub — CHANGES_REQUESTED** [review]

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/ServiceCatalog/Listing/Onboarding.jsx (hunk 1 above)
Comment (Jayanie) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/ServiceCatalog/Listing/Onboarding.jsx]: Fixed with 34700c2a7356a5d035adb21e29659a476a6a07b7

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/ServiceCatalog/Listing/Onboarding.jsx (hunk 2 above)
Comment (Jayanie) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/ServiceCatalog/Listing/Onboarding.jsx]: Fixed with 34700c2a7356a5d035adb21e29659a476a6a07b7

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/ServiceCatalog/Listing/Onboarding.jsx (hunk 1 above)
Comment (Jayanie) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/ServiceCatalog/Listing/Onboarding.jsx]: Fixed with 2e937d4af09e1e8547f26dc5bffb8962e6156a3e

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/ServiceCatalog/Listing/Onboarding.jsx (hunk 2 above)
Comment (Jayanie) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/ServiceCatalog/Listing/Onboarding.jsx]: Fixed with 2e937d4af09e1e8547f26dc5bffb8962e6156a3e

**Review by tharikaGitHub — APPROVED** [review]

**Comment by malinthaprasan** [comment]
Not required as this was changed in the master branch

**Pull request closed without merging**